router/middleware/header.go

```diff
@@ -51,12 +51,9 @@ func Secure(c *gin.Context) {
 	c.Header("X-Frame-Options", "DENY")
 	c.Header("X-Content-Type-Options", "nosniff")
 	c.Header("X-XSS-Protection", "1; mode=block")
-
-	// Also consider adding Content-Security-Policy headers
+	// TODO: consider adding Content-Security-Policy headers
 	// c.Header("Content-Security-Policy", "script-src 'self' https://cdnjs.cloudflare.com")
-	if c.Request.TLS != nil {
-		c.Header("Strict-Transport-Security", "max-age=31536000")
-	}
+	c.Header("Strict-Transport-Security", "max-age=63072000; includeSubDomains; preload")
```

Comment:
Since you're changing the max-age here, does this need to be configurable somehow?

Reply:
Cool. That makes sense. Making it configurable is probably overkill then.